src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/PolicyTests.ps1

```diff
@@ -35,7 +35,7 @@ function Test-PolicyDefinitionCRUD
 
 	New-AzureRMPolicyDefinition -Name test2 -Policy "{""if"":{""source"":""action"",""equals"":""blah""},""then"":{""effect"":""deny""}}"
 	$list = Get-AzureRMPolicyDefinition
-	Assert-AreEqual 2 @($list).Count
+	#Assert-AreEqual 2 @($list).Count
```

Comment:
but we need to make sure this passes in playback mode ,so its fine to comment it in record mode,assuming we get more than 2 as the listcount,however do not comment in playback mode

---
In reply to: [126340017](https://github.com/Azure/azure-powershell/pull/4257#discussion_r126340017) [](ancestors = 126340017)